addons/website_event/tests/test_event_internals.py

```diff
@@ -115,6 +115,40 @@ def test_process_attendees_form(self):
                 (0, 0, {'question_id': self.event_question_3.id, 'value_text_box': 'Free Text'})]}
         ])
 
+    def test_process_attendees_form_no_tickets(self):
+        """Check that registering with no ticket works."""
+        event = self.env['event.event'].create({
+            'name': 'Event Update Type',
```

Comment:
Let's just go for "Test Event" ? I feel like "Event Update Type" is being copy/pasta'ed everywhere ^^